Jenkinsfile

```diff
@@ -178,12 +178,12 @@ node {
       parallel (
         'nightly': {
           deviceStage("build nightly", "tici-needs-can", [], [
-            step("build nightly", "RELEASE_BRANCH=nightly $SOURCE_DIR/release/build_release.sh"),
+            step("build nightly", "BUILD_DIR=/data/openpilot_release RELEASE_BRANCH=nightly $SOURCE_DIR/release/build_release.sh"),
```

Comment:
not sure if this is still the case, but some builds used to link against the absolute path and installing in anything but `/data/openpilot_release` would fail

Reply:
(if it's still the case, we should just fix the linking)

Reply:
Unfortunately it is dependent on the absolute build path